packaging/wix/make_installer.ps1

```diff
@@ -0,0 +1,5 @@
+python configure.py
+pandoc -f markdown -t rtf ..\..\LICENSE.md -s -o LICENSE.rtf
```

Comment:
The more I think about it... maybe it is better to version `LICENSE.rtf`? The thing is, I'd like to build a WiX installer in AppVeyor on each tagged commit / release. AppVeyor has Wix 3.11 (as of now) and Python installed by default, but no `pandoc`.